etc/scripts/scancode.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please do not name this scancode ;) that's confusing!

Reply:
Suggest the better name.

Reply:
`deps_info.py`